src/main/java/com/epam/university/java/core/task010/Task010Impl.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.university.java.core.task010;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Task010Impl implements Task010 {
+    @Override
+    public Map<String, Integer> countWordNumbers(File source) {
+        String text = "";
+        try {
+            Scanner scanner = new Scanner(source);
+            while (scanner.hasNext()) {
+                text += scanner.nextLine();
+            }
+            ;
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
You'd better avoid using the `e.printStackTrace()` in favor of using loggers. And as far as I can see, the exception is swallowed that's not good btw.